acs-apiclient/AcsApiClientConfig.cs

```diff
@@ -87,14 +87,14 @@ public class AcsApiClientConfig
         internal readonly string ConsumerSecret;
 
         /// <summary>
-        /// Gets or sets the Access token
+        /// A token received upon authenticating
```

Comment:
💡 "A token received upon completing the entire login process. This will be used for access to the ForeSee services"